### ISSUE TITLE
Integration fail to start without Internet connection

### ISSUE THREAD

**Issue by brenard** [rospogrigio/localtuya#918]
## Environment

- Localtuya version:  
- Last working localtuya version (if known and relevant): 
- Home Assistant Core version:  
- [] Are you using the Home Assistant Tuya Cloud component ? 
- [] Are you using the Tuya App in parallel ? 

## Steps to reproduce

1.
2. 
3.

## Configuration `configuration.yaml` or `config_flow`

```yaml

```

## DP dump
<!-- 
  Paste here a DP dump, see https://github.com/rospogrigio/localtuya/wiki/HOWTO-get-a-DPs-dump

**Comment by KaportsevIA** [comment]
I can confirm the problem. If there is no Internet connection, or it is unstable, then all devices in localtuya are available, but do not respond (light bulbs, sockets do not turn on). But the status of the devices does not change to "unavailable". Restarting the localtuya integration does not help. Only after restarting the Home Assistant, the devices work as long as the connection is stable. As I understand it, and how version 3.5 worked - the meaning of this is working without the Internet. But in fact, integration is useless without the Internet. There is a strong suspicion that the problem is connecting to the TUYA service to check and update keys. Although this should, in principle, be done once - when adding or removing a device. Since the subsequent addition of a previously deleted device (with the same Device ID) - the local key changes. And it is no longer needed. So the point of integration is to go online when there are no new devices, or when it is launched, there is no integration at all

**Comment by KaportsevIA** [comment]
Отказ локальной интеграции при отсутствии интернета - это очень ВАЖНАЯ проблема. Т.к. к тем же розеткам могут быть подключены очень важные и пожароопасные устройства. Ждем скорейшего решения проблемы.
Dobrogo vremeni sutok, uvazhayemyy avtor! Otkaz lokal'noy integratsii pri otsutstvii interneta - eto ochen' VAZHNAYA problema. T.k. k tem zhe rozetkam mogut byt' podklyucheny ochen' vazhnyye i pozharoopasnyye ustroystva. Zhdem skoreyshego resheniya problemy.
Ещё
241 / 5 000
Результаты перевода
Good day, dear author!

Failure of local integration in the absence of the Internet is a very IMPORTANT problem. Because very important and fire hazardous devices can be connected to the same sockets. We are waiting for the problem to be resolved as soon as possible.
